README.md

```diff
@@ -22,6 +22,19 @@ C++ client for [ClickHouse](https://clickhouse.com/).
 * Int128
 * UUID
 
+## Requirement
+
+You need gcc cmake libabsl-dev to build this project.
```

Comment:
1) Should work without `libabsl-dev`
2) gcc version 7 at least (clang >= 6 also supperted, after #131 MSVC should work also).

Reply:
To build with openssl libssl-dev is also needed.

Reply:
See https://github.com/ClickHouse/clickhouse-cpp/blob/master/.github/workflows/linux.yml

Reply:
Hello world does not need https i think. or am i wrong?

I just finish running the hello world for me ,so it's time to share how to run example code , may help who want to play .

Reply:
> Hello world does not need https i think

that readme is for the clickhouse-cpp, why not make it clear for all the cases?